src/Product/Attributes/Adult.php

```diff
@@ -59,4 +59,25 @@ public static function get_value_type(): string {
 	public static function get_input_type(): string {
 		return AdultInput::class;
 	}
+
+	/**
+	 * Returns the attribute name
+	 *
+	 * @return string
+	 */
+	public static function get_name(): string {
+		return __( 'Adult', 'google-listings-and-ads' );
+	}
+
+	/**
+	 * Returns the attribute sources
+	 *
+	 * @return array
+	 */
+	public static function get_sources(): array {
+		return [
+			'yes' => __( 'Yes', 'google-listings-and-ads' ),
+			'no'  => __( 'No', 'google-listings-and-ads' ),
```

Comment:
The API expects a boolean type and doesn't support `yes/no` strings. Are we going to keep all mapping values as strings and only convert to the right type when we are setting the attributes?

Reply:
With API I take as granted you mean Google API.

> Are we going to keep all mapping values as strings and only convert to the right type when we are setting the attributes?

Yes, that was my initial idea since Frontend will store it as strings. We can do also "true" and "false"  it that makes more sense and it's easier (maybe even automatic) to convert to boolean.

Reply:
However, it would be more convenient for me to wait til the future PR to load the attributes in the Frontend selector to verify this and bet for the most elegant solution.

Reply:
Yes I meant the Google API. `yes/no` is good, it's what WC also uses for boolean settings that are stored in the options table. Was mainly curious, no need to change anything here.